README.md

```diff
@@ -84,7 +84,12 @@ Tests are an important part of the development and evolution of this project,
 and new contributions are expected to include tests for any functionality
 change.  To run the tests, pass the `test` verb to the `bootstrap` script:
 
-    ./Utilities/bootstrap --build-tests test
```

Comment:
I see! Yeah, I'll send out a separate pull request for the documentation, and simply include the new, still unused Foundation parameter in this one.